InvoiceProcessor.Tests/Application/Queries/Outbox/GetPendingProcessTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using InvoiceProcessor.Domain.Entities;
+using InvoiceProcessor.Domain.Enums;
+using InvoiceProcessor.Domain.Interfaces.Outbox;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using static InvoiceProcessor.Application.Queries.Outbox.GetPendingProcess.GetPendingProcess;
+
+namespace InvoiceProcessor.Tests.Application.Queries.Outbox
+{
+    public class GetPendingProcessTests
+    {
+        private readonly Mock<IOutboxStorage> _outboxStorageMock;
+
+        public GetPendingProcessTests()
+        {
+            _outboxStorageMock = new Mock<IOutboxStorage>();
+        }
+
+        [Fact]
+        public async Task GetPendingProcess_Should_Return_Empty_When_Pending_Process_Unavailable()
+        {
+            // Arrange
+            List<OutboxItem> outboxItems = null;
+            List<Model> models = null;
```

Comment:
When there is no pending invoice, it returns an empty List not null. And when you want to pass null, you directly can pass null insisted of creating another variable.